### PR TITLE
Update how-to-monitor-nginx-the-essential-guide.md

### PR DESCRIPTION
**Pull request by oleersoy**

### CODE REVIEW

File: guides/how-to-monitor-nginx-the-essential-guide.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ While nginx's open source variant (nginx F/OSS, or "plain 'ol nginx") is the mos
 
 While there's a body of conventional wisdom and some basic material on the web, we found no definitive resource on the subject.
 
-At Scalyr, our approach to monitoring is based on years of frontline operational experience.  Good monitoring will help you more detect problems more quickly, spot incipient / slow-burn issues before they become full-scale outages, and ultimately save you (and your users) headaches.
+At Scalyr, our approach to monitoring is based on years of frontline operational experience.  Good monitoring will help you detect problems more quickly, spot incident / slow-burn issues before they become full-scale outages, and ultimately save you (and your users) headaches.
```
Comment (czerwingithub) [inline guides/how-to-monitor-nginx-the-essential-guide.md]: Thanks for removing the extra `more`, but `incipient` is the right word here.  Can you change that back and we can commit?

Comment (oleersoy) [inline guides/how-to-monitor-nginx-the-essential-guide.md]: Yup I realized I got a little to correction happy as I kept reading.  Great article BTW.  Any idea how I change it back.  Would it be easier to just decline the PR and change it manually?

Comment (oleersoy) [inline guides/how-to-monitor-nginx-the-essential-guide.md]: Actually I think I got it - look OK now?

**Comment by czerwingithub** [comment]
Thanks again for doing this.  We appreciate it.. and glad you enjoyed the guide.

**Comment by noahlh** [comment]
This has been pushed to the production Community site.

**Pull request merged**